File: Python/Ejercicio 6/main.py
nua = 1
max = 0
min = 0

while (nua != 0):
    nua = int(input("ingrese un numero"))
    if (max == 0 & min ==0):
        max=nua
        min=nua
    if (nua > max):
        max = nua
    if (nua < min):
        min = nua

print ("el numero mas bajo es: ", min, "el numero mas alto es: ", max)

File: C++/Ejercicio 6/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int numeroA, maximo=0, minimo =0;

    do
    {
        cout << " ingrese un numero " << endl;
        cin >> numeroA;
        if (numeroA > maximo)
        {
            maximo = numeroA;
        }
        if (numeroA < minimo)
        {
            minimo = numeroA;
        }

    }while (numeroA != 0);

    cout << "el numero mas alto es " << maximo << endl;
    cout << "el numero mas bajo es " << minimo << endl;




}
